src/dom/dom.js

```diff
@@ -359,16 +516,42 @@ p5.prototype.createP = function (html = '') {
 };
 
 /**
- * Creates a `&lt;span&gt;&lt;/span&gt;` element in the DOM with given inner HTML.
+ * Creates a `&lt;span&gt;&lt;/span&gt;` element. It's commonly used as a
+ * container for other elements.
+ *
+ * The parameter `html` is optional. It accepts a string that sets the
+ * inner HTML of the new `&lt;span&gt;&lt;/span&gt;`.
  *
  * @method createSpan
- * @param  {String} [html] inner HTML for element created
- * @return {p5.Element} pointer to <a href="#/p5.Element">p5.Element</a> holding created node
+ * @param  {String} [html] inner HTML for the new `&lt;span&gt;&lt;/span&gt;` element.
+ * @return {p5.Element} new <a href="#/p5.Element">p5.Element</a> object.
  * @example
- * <div><code>
- * let span = createSpan('this is some text');
- * span.position(0, 0);
- * </code></div>
+ * <div>
+ * <code>
+ * function setup() {
+ *   background(200);
+ *
+ *   let span = createSpan('p5*js');
+ *   span.position(25, 35);
+ *
+ *   describe('A gray square with the text "p5*js" written in its center.');
+ * }
+ * </code>
+ * </div>
+ *
+ * <div>
+ * <code>
+ * function setup() {
+ *   background(200);
+ *
+ *   // Create an h3 element within the span.
+ *   let span = createSpan('<h3>p5*js</h3>');
```

Comment:
This works but I also wonder if there's a way to illustrate the difference with a div here. Maybe if you have a div that already has some text, you could add a span with red text into the middle of its contents?